guice-benchmark/src/strategies/output/OutputToFile.java

```diff
@@ -33,6 +33,7 @@ public OutputStream getOutputStream(Config config) {
 		} catch (FileNotFoundException e) {
 			// Auto-generated catch block
 			e.printStackTrace();
+			throw new Exception("Error creating FileOutputStream");
```

Comment:
Change the exception to RuntimeException ("Error creating FileOutputStream",e), then the method does not have to declare "throws Exception" in the method signature.  Then you can undo the interface change